### PR TITLE
scan: update submittxmodal config

### LINKED ISSUE #1487: Scan: implement gas amount's config of send tx
Let's make the config file that defines how much `gas` and `feeAmount` are used for each message.

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1487, #1603, #1605, #1606, #1607

### CODE REVIEW

File: scan/src/components/modal/submitTx/SendMsg.re [hunk 1]
```diff
@@ -9,7 +9,10 @@ let make = (~setMsgsOpt) => {
         let%Opt toAddressValue = toAddress.value;
         let%Opt amountValue = amount.value;
         Some([|
-          TxCreator.Send(toAddressValue, {amount: amountValue |> string_of_int, denom: "uband"}),
+          TxCreator.Send(
+            toAddressValue,
+            {amount: amountValue *. 1e6 |> Js.Float.toString, denom: "uband"},
```
Comment (evilpeach) [inline scan/src/components/modal/submitTx/SendMsg.re]: here as well

File: scan/src/components/modal/submitTx/DelegateMsg.re [hunk 2]
```diff
@@ -11,7 +11,7 @@ let make = (~setMsgsOpt) => {
         Some([|
           TxCreator.Delegate(
             validatorValue,
-            {amount: amountValue |> string_of_int, denom: "uband"},
+            {amount: amountValue *. 1e6 |> Js.Float.toString, denom: "uband"},
```
Comment (evilpeach) [inline scan/src/components/modal/submitTx/DelegateMsg.re]: Let's put the parsing process into parse function.

File: scan/src/components/modal/submitTx/RedelegateMsg.re [hunk 3]
```diff
@@ -14,7 +14,7 @@ let make = (~setMsgsOpt) => {
           TxCreator.Redelegate(
             srcValidatorValue,
             dstValidatorValue,
-            {amount: amountValue |> string_of_int, denom: "uband"},
+            {amount: amountValue *. 1e6 |> Js.Float.toString, denom: "uband"},
```
Comment (evilpeach) [inline scan/src/components/modal/submitTx/RedelegateMsg.re]: here as well

File: scan/src/components/modal/submitTx/UndelegateMsg.re [hunk 4]
```diff
@@ -11,7 +24,7 @@ let make = (~setMsgsOpt) => {
         Some([|
           TxCreator.Undelegate(
             validatorValue,
-            {amount: amountValue |> string_of_int, denom: "uband"},
+            {amount: amountValue *. 1e6 |> Js.Float.toString, denom: "uband"},
```
Comment (evilpeach) [inline scan/src/components/modal/submitTx/UndelegateMsg.re]: here as well

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**